test/decode.js

```diff
@@ -34,6 +34,22 @@ describe('decode', function() {
   it('parses url-encoded payload into a buffer', function() {
     var buffer = dataUri.decode('data:,hello%20world');
     expect(buffer.toString('ASCII')).to.equal('hello world');
+    buffer = dataUri.decode('data:;base64,aGVsbG8gd29ybGQ=');
+    expect(buffer.toString('ASCII')).to.equal('hello world');
+    buffer = dataUri.decode('data:;charset=iso-8859,hello%20world');
+    expect(buffer.toString('ASCII')).to.equal('hello world');
+  });
```

Comment:
Please create a new test for each decode+expect pair and provide a descriptive name explaining the difference between the test cases (input data).